en/deploy-tidb-enterprise-edition.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Deploy TiDB Enterprise Edition in Kubernetes
+summary: Learn how to deploy TiDB Enterprise Edition in Kubernetes.
+---
+
+# Deploy TiDB Enterprise Edition in Kubernetes
+
+This document describes how to deploy TiDB Cluster Enterprise Edition and related tools in Kubernetes. TiDB Enterprise Edition has the following features:
+
+* Enterprise-level best practices
+* Enterprise-level service support
+* Fully enhanced security features
+
+## Prerequisites
+
+* TiDB Operator is [deployed](deploy-tidb-operator.md).
+  
+## Deploy method
+
+Currently, the difference between the deployment of TiDB Operator Enterprise Edition and Community Edition is mainly reflected in the image naming. Compared with Community Edition, the image of Enterprise Edition has an extra `-enterprise` suffix.
+
+```yaml
+spec:
+  version: v4.0.2
+  ...
+  pd:
+    baseImage: pingcap/pd-enterprise
+  ...
+  tikv:
+    baseImage: pingcap/tikv-enterprise
+  ...
+  tidb:
+    baseImage: pingcap/tidb-enterprise
+  ...
+  tiflash:
+    baseImage: pingcap/tiflash-enterprise
+  ...
+  pump:
+    baseImage: pingcap/tidb-binlog-enterprise
+  ...
+  ticdc:
+    baseImage: pingcap/ticdc-enterprise
+```
+
+If you are deploying a new cluster, refer to [Configure a TiDB Cluster in Kubernetes](configure-a-tidb-cluster.md) to configure `tidb-cluster.yaml` and the enterprise edition image as described above, and run the `kubectl apply -f tidb-cluster.yaml -n ${namespace }` command to deploy TiDB Enterprise Edition cluster and related tools.
```

Comment:
```suggestion
If you are deploying a new cluster, refer to [Configure a TiDB Cluster in Kubernetes](configure-a-tidb-cluster.md) to configure `tidb-cluster.yaml` and the enterprise edition image as described above, and run the `kubectl apply -f tidb-cluster.yaml -n ${namespace}` command to deploy the TiDB Enterprise Edition cluster and related tools.
```

Reply:
Suggest using a list for the following section: deploy a new cluster, switch from Community Edition.